apisix/plugins/consumer-restriction.lua

```diff
@@ -53,6 +53,38 @@ local schema = {
                 rejected_code = {type = "integer", minimum = 200, default = 403}
             },
             required = {"whitelist"},
+        },
+        {
+            title = "allowed_methods",
+            properties = {
+                type = {
```

Comment:
Can we extra this part and share across the components with different titles?

Reply:
Sorry, I'm not sure to understand what you mean by extra

Reply:
Sorry, it is my typo. It should be `extract`.